src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java

```diff
@@ -88,8 +92,8 @@ public void verifyMappingsAreValid() throws MappingInitializationException {
         for (Map.Entry<String, String> entry : getMappingWithUnmappedColumns(false).entrySet()) {
             String sfdcName = entry.getValue();
             if(StringUtils.hasText(sfdcName)) {
-                final Field f = getClient().getField(sfdcName);
-                if (f == null)
+                final HashSet<Field> fieldSet = getClient().getField(sfdcName);
+                if (fieldSet == null | fieldSet.isEmpty())
```

Comment:
It is!

But don't let that distract you.  My changes were based off the 33.0 
branch 'cause that's the only one I can test with.  I'm unsure how to 
test v34 against my orgs.

I can fix that, but my bigger question is, though it works for what I 
need, are the maintainers OK with its implementation?  I figured 
creating a pull request was the easiest way to share the code.